net/Socket.hpp

```diff
@@ -1443,21 +1518,13 @@ class StreamSocket : public Socket,
         if (closed)
         {
             LOG_TRC("Closed. Firing onDisconnect.");
-            _closed = true;
             _socketHandler->onDisconnect();
-        }
-
-        if (_closed)
+            setClosed(disposition);
+        } else if (!isOpen()) {
```

Comment:
Formatting. (I know, you are still working out the clang-format setup, but pointing out anyway.)

Reply:
No, I don't still figuring out my clang-format setup. But yes, I apply it manually for bigger edited code chunks only.